README.md

```diff
@@ -2,31 +2,260 @@
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Learn about the Java `Random` class
 
 ## Introduction
 
-A 1-2 sentence summary of what will be covered.
+Java has yet another class to help generate random numbers a little easier
+called the `Random` class!
 
-## Topic 1
+The **Random Class** is part of the package `java.util` and contains methods to
+generate different types of random numbers. To use the `Random` class, we must
+import it using the `import` statement like we saw in the Packages lesson:
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+```java
+import java.util.Random;
+```
 
-## Topic 2
+Notice we didn't have to import the Math class in the previous lesson. This is
+because the Math class is available in the `java.lang` package. If we remember
+from the Packages lesson, we do not need to import the `java.lang` package
+since it is in the default package.
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+Once we have imported the `Random` class, we can use it anywhere in our class by
+instantiating an object of that type:
 
-## Conclusion
+```java
+Random random = new Random();
+```
 
-A short one or two paragraph summary of the contents of the lessons, recapping
-the learning goals.
+Now that we have instantiated our `Random` instance, let's put it to use by
+using some of its methods!
 
-## Resources
+## `nextBoolean()`
 
-- [Resource Link 1](example.com)
-- [Resource Link 2](example.com)
+If we want a random `boolean` value, we can use the `Random` class' method
+`nextBoolean()` to return a pseudorandom `boolean` value of true or false. For
```

Comment:
Are students expected know what pseudorandom is when they do this lesson?

Reply:
Added an explanation.